packages/arb-avm-cpp/data_storage/include/data_storage/util.hpp

```diff
@@ -17,6 +17,8 @@
 #ifndef ARB_AVM_CPP_UTIL_HPP
 #define ARB_AVM_CPP_UTIL_HPP
 
+#include "cpruningmode.h"
```

Comment:
I think we want to avoid `data_storage` depending on `cavm`. It should work to just move `cpruningmode.h` into `data_storage` I think

Reply:
:+1: